x/pki/handler_test.go

```diff
@@ -2266,6 +2266,57 @@ func TestHandler_AddPkiRevocationDistributionPoint_PAIWithStringVidPid(t *testin
 	require.NoError(t, err)
 }
 
+func TestHandler_AddPkiRevocationDistributionPoint_DataURLNotUnique(t *testing.T) {
```

Comment:
Please also add a test where issuerSubjectKeyId and DataUrl are the same, but VIDs are different. Expected behaviour - no errors.